jmix-flowui/flowui-kit/src/main/resources/META-INF/frontend/theme/material/tabsheet/jmix-tabsheet-styles.js

```diff
@@ -1,4 +1,4 @@
-// CAUTION: copied from @vaadin/tabsheet/theme/material/vaadin-tabsheet-style.js [since Vaadin 24.1.10]
+// CAUTION: copied from @vaadin/tabsheet/theme/material/vaadin-tabsheet-style.js [since Vaadin 24.2.1]
```

Comment:
Since **Vaadin 24.2.1 looks** incorrect. We copied a long time ago. Let's use `last update Vaadin 24.2.1` as in the other "old" files.